spec/views/users/emails/verify.html.erb_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe 'users/emails/verify.html.erb' do
+  let(:email) { 'foo@bar.com' }
+  before do
+    allow(view).to receive(:email).and_return(email)
+    @resend_email_confirmation_form = ResendEmailConfirmationForm.new
+  end
+
+  it 'has a localized title' do
+    expect(view).to receive(:title).with(t('titles.verify_email'))
+
+    render
+  end
+
+  it 'has a localized header' do
+    render
+
+    expect(rendered).to have_selector('h1', text: t('headings.verify_email'))
+  end
+
+  it 'contains link to resend confirmation page' do
+    render
+
+    expect(rendered).to have_button(t('links.resend'))
+  end
+
+  context 'when enable_load_testing_mode? is true and email address found' do
+    before do
+      allow(FeatureManagement).to receive(:enable_load_testing_mode?).and_return(true)
+      email_address = instance_double(EmailAddress, confirmation_token: 'some_token')
+      allow(EmailAddress).to receive(:find_with_email).with(email).and_return(email_address)
```

Comment:
ditto, use a factory here
```suggestion
      craete(:email_address, confirmation_token: 'some_token')
```